datameta/api/files.py

```diff
@@ -169,6 +172,8 @@ def update_file(request: Request) -> HTTPOk:
         raise HTTPNotFound(json=None) # 404
     if db_file.user_id != auth_user.id:
         raise HTTPForbidden(json=None) # 403
+    if not db_file.name:
```

Comment:
Here you are just checking if the entry in the database is wrong, right? This should actually never happen, and if it does we shouldn't throw an error here because the user might want to correct this invalid field.
I think this check should be move to line 186.

Reply:
Right, thanks @KerstenBreuer.